gatsby/content/projects/bridges/matrix-bifrost.mdx

```diff
@@ -3,18 +3,29 @@ layout: projectimage
 title: matrix-bifrost
 categories: 
  - bridge
-description: bridging with a variety of backends including libpurple and xmpp.js
+description: bridging with xmpp.js
```

Comment:
The old description, while probably not the most wieldly, was more apt.

Reply:
Sorry missed that the matrix.org bridge still used libpurple. The aria-net bridge don't use libpurple any more